src/main/java/com/serdar/calculator/service/ArithmeticServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.serdar.calculator.service;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import com.serdar.calculator.entity.CustomRequest;
+import com.serdar.calculator.utils.Utils;
+
+@Service
+public class ArithmeticServiceImpl implements ArithmeticService {
+
+    private final List<Character> operations = Arrays.asList(new Character[]{'+', '-'});
+
+    @Override
+    public BigDecimal calculate(CustomRequest customRequest) {
+        String firstNumber = customRequest.firstNumber;
+        String secondNumber = customRequest.secondNumber;
+        Character operation = customRequest.operation;
+
```

Comment:
Remember to add all bad request exceptions in here. Related to this [PBI](https://serdarakyol.atlassian.net/browse/LEARN-95)